JavaScript/Topics/17.jQuery/README.md

```diff
@@ -0,0 +1,138 @@
+# What is jQuery ?
+
+  Query is a fast, small, cross-platform and feature-rich JavaScript library. It is designed to simplify the client-side scripting of HTML.jQuery is a lightweight, "write     less, do more", JavaScript library.jQuery takes a lot of common tasks that require many lines of JavaScript code to accomplish, and wraps them into methods that you can call with a single line of code.jQuery also simplifies a lot of the complicated things from JavaScript, like AJAX calls and DOM manipulation.
+
+  The purpose of jQuery is to make it much easier to use JavaScript on your website.
+
+  The jQuery library contains the following features:<br>
+  --> HTML/DOM manipulation<br>
+  --> CSS manipulation<br>
+  --> HTML event methods<br>
+  --> Effects and animations<br>
+  --> AJAX<br>
+  --> Utilities<br>
+
+# Is jQuery a programming language?
+  jQuery is not a programming language but a well-written JavaScript code. It is used to traverse documents, event handling, Ajax interaction, and Animation.
+
+# What is the difference between JavaScript and jQuery?
+  The simple difference is that JavaScript is a language while jQuery is a built-in library built for JavaScript. jQuery simplifies the use of JavaScript language.  
+
+# Why jQuery is required ?
+  --> It is very fast and extensible.<br>
+  --> It facilitates the users to write UI related function codes in minimum possible lines.<br>
+  --> It improves the performance of an application.<br>
+  --> Browser's compatible web applications can be developed.<br>
+  --> It uses mostly new features of new browsers.<br>
+  --> So, you can say that out of the lot of JavaScript frameworks, jQuery is the most popular and the most extendable. Many of the biggest companies on the web use jQuery.<br>
+
+  Some of these companies are:<br>
+  --> Microsoft<br>
+  --> Google<br>
+  --> IBM<br>
+  --> Netflix<br>
+
+# What should you know before starting to learn jQuery?
+  It is always advised to a fresher to learn the basics of web designing before starting to learn jQuery. He should learn HTML, CSS and JavaScript first. But, if you belong to a technical background, it is up to you.
+
+# what is jQuery History?
+  jQuery was first released in January 2006 by John Resig at BarCamp NYC. It is currently headed by Timmy Wilson and maintained by a team of developers. Nowadays, jQuery is widely used technology. Most of the websites are using jQuery.
+
+  jQuery Release History-
+
```

Comment:
Just provide first release version of jQuery, the documentation should be precise:)